editions/webdav/tiddlywiki.info

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
why are there 2 different editions?

Reply:
This is the new one - the other was already there.

Reply:
ok. As I wrote. Please create a second PR for the other one.